shap/plots/dependence.py

```diff
@@ -138,6 +142,15 @@ def convert_name(ind):
     if categorical_interaction and clow != chigh:
         bounds = np.linspace(clow, chigh, chigh - clow + 2)
         color_norm = matplotlib.colors.BoundaryNorm(bounds, colors.red_blue.N)
+        
+    # optionally add jitter to feature values
+    if x_jitter > 0:
+        if x_jitter > 1: x_jitter = 1
+        xvals = list(set(xv))
```

Comment:
```suggestion
        xvals = np.unique(xv)
```
Per @slundberg 's recommendation